cvat/apps/consensus/merge_consensus_jobs.py

```diff
@@ -0,0 +1,102 @@
+from typing import List, Dict
+import datumaro as dm
+import django_rq
+from django.conf import settings
+from datumaro.components.operations import IntersectMerge
+from cvat.apps.quality_control.quality_reports import JobDataProvider
+from cvat.apps.engine.models import Job, JobType
+from cvat.apps.dataset_manager.bindings import import_dm_annotations
+from rest_framework import status
+from rest_framework.response import Response
+from cvat.apps.dataset_manager.task import patch_job_data, PatchAction
+from cvat.apps.engine.utils import get_rq_lock_by_user, get_rq_job_meta, define_dependent_job, process_failed_job
+from cvat.apps.engine.serializers import RqIdSerializer
+from django.utils import timezone
+from django.db import transaction
+
+def get_consensus_jobs(task_id: int):
```

Comment:
```suggestion
def get_consensus_jobs(task_id: int) -> dict[int, list[int]]:
```

Please add other missing type annotations in functions in this file (except for some django stuff, like `request`).